services/api/package.json

```diff
@@ -27,16 +27,18 @@
     "@koa/router": "^9.3.1",
     "@sentry/node": "^6.2.1",
     "bcrypt": "^5.0.0",
+    "bytes": "^3.1.0",
```

Comment:
mmm do we really need to? is worth for the extra dependency ? In google cloud its formatted (thats what happens in my mind)

Reply:
well koa logger was showing `300B` `4KB` etc so I was trying to represent that. This is just for development so I don't mind a bit of extra fluff

Reply:
oh unless you're saying because I'm over-formatting the message in production as well

I think maybe I need a less on how instrumentation actually works

Reply:
maybe I'm thinking about this the wrong way... what about this:

```
const formatters = {
  // https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry
  gcloud: // <-- revert this back to what it was before I messed with it
  dev: // <-- new simple thing with pretty colors
};
```

then later:

```
    httpRequestFormat: ENV_NAME === 'development' ? formatters.dev : formatters.gcloud,
```

Reply:
we could then remove `pino-prettier` dependency that way too because we no longer need to transform structured JSON logging on dev, we simply don't do it... seems a lot cleaner